docs/notebooks/verification_gridded_cosmo.ipynb

```diff
@@ -0,0 +1,3105 @@
+    "        # Plot ground truth\n",
+    "        im0 = axes[0].pcolormesh(\n",
+    "            lons,\n",
+    "            lats,\n",
+    "            ds_var.values,\n",
+    "            transform=ccrs.PlateCarree(),\n",
+    "            vmin=vmin,\n",
+    "            vmax=vmax,\n",
+    "            cmap=\"viridis\",\n",
+    "            shading=\"auto\",\n",
+    "        )\n",
+    "        axes[0].set_title(\"Ground Truth\")\n",
+    "\n",
+    "        # Plot order depends on NWP availability\n",
+    "        plot_idx = 1\n",
+    "        if ds_nwp_time is not None and var in ds_nwp_time:\n",
+    "            axes[plot_idx].pcolormesh(\n",
+    "                lons,\n",
+    "                lats,\n",
+    "                ds_nwp_var.values,\n",
+    "                transform=ccrs.PlateCarree(),\n",
+    "                vmin=vmin,\n",
+    "                vmax=vmax,\n",
+    "                cmap=\"viridis\",\n",
+    "                shading=\"auto\",\n",
+    "            )\n",
+    "            axes[plot_idx].set_title(\"NWP Model Prediction\")\n",
+    "            plot_idx += 1\n",
+    "\n",
+    "        # Plot ML prediction (always last)\n",
+    "        axes[plot_idx].pcolormesh(\n",
+    "            lons,\n",
+    "            lats,\n",
+    "            ds_ml_var.values,\n",
+    "            transform=ccrs.PlateCarree(),\n",
+    "            vmin=vmin,\n",
+    "            vmax=vmax,\n",
+    "            cmap=\"viridis\",\n",
+    "            shading=\"auto\",\n",
+    "        )\n",
+    "        axes[plot_idx].set_title(\"ML Model Prediction\")\n",
+    "\n",
+    "        # Add common features to all plots\n",
+    "        for ax in axes:\n",
+    "            ax.coastlines(resolution=\"50m\")\n",
+    "            ax.add_feature(cfeature.BORDERS, linestyle=\"-\", alpha=0.7)\n",
+    "            gl = ax.gridlines(\n",
+    "                draw_labels=True, dms=True, x_inline=False, y_inline=False\n",
+    "            )\n",
+    "            gl.top_labels = False\n",
+    "            gl.right_labels = False\n",
+    "\n",
+    "        # Add colorbar\n",
+    "        cbar_ax = fig.add_axes([0.2, -0.05, 0.6, 0.05])\n",
+    "        cbar = fig.colorbar(im0, cax=cbar_ax, orientation=\"horizontal\")\n",
+    "        cbar.set_label(VARIABLE_UNITS[var])\n",
+    "\n",
+    "        # Adjust subplot spacing\n",
+    "        plt.subplots_adjust(bottom=0.15, hspace=0.05, wspace=0.05)\n",
+    "\n",
+    "        if elapsed_forecast_duration is not None:\n",
+    "            title = f\"{var} at {str(time_selected.dt.date.values)} - {time_selected.dt.hour.values:02d} UTC and Elapsed Forecast Duration +{int(elapsed_forecast_duration.values / 1e9 / 3600)}h\"\n",
+    "            save_name = f\"map_{var}_leadtime_{int(elapsed_forecast_duration.values):02d}\"\n",
+    "        else:\n",
+    "            title = f\"{var} at {str(time_selected.dt.date.values)} - {time_selected.dt.hour.values:02d} UTC\"\n",
```

Comment:
We need to put some thought about what plot titles to include here, and which one should be in tex. I am generally in favor of having as little titles as possible from matplotlib, and rather adding these in the paper. For example, the variable and time I think should be part of the figure caption in tex rather than printed in the figure. 

It is still very useful to have these titles here in the notebook, so maybe we should just have some way to turn them off? Or should I just comment out the title when making the plot to import in the paper?

Reply:
i have written a small routine that before saving a plot to disk it checks whether there is a `plt.suptitle` or if not maybe a `plt.title` is available. if so, these are removed from the figure and you can add the titles directly in latex.